zap/src/main/java/org/zaproxy/zap/extension/ascan/ActiveScan.java

```diff
@@ -234,6 +238,18 @@ public void hostProgress(int id, String hostAndPort, String msg, int percentage)
         }
     }
 
+    @Override
+    public void filteredMessage(HttpMessage msg, String reason) {
+        try {
+            filterMessageTableModel.addScanResult(SessionStructure.getNodeName(msg), reason);
+        } catch (URIException e) {
+            log.error(
```

Comment:
`warn` or `debug` (this shouldn't really happen, given that the message was already created processed).

Reply:
ok